libs/blocks/global-navigation/global-navigation.css

```diff
@@ -421,15 +431,17 @@ header.global-navigation {
   padding: 12px 0;
 }
 
-.feds-signIn-dropdown li > a {
+.feds-signIn-dropdown li > a,
+.feds-signIn-dropdown .feds-signIn {
   display: block;
   color: var(--feds-color-link--light);
   padding: 6px 32px;
 }
 
-.feds-signIn-dropdown li > a:hover {
+.feds-signIn-dropdown li > a:hover,
+.feds-signIn-dropdown li > button:hover {
```

Comment:
I see this just uses `button`, while L435 uses `.feds-signIn`. Do they need to be different?